### PR TITLE
refactoring ghgvc function on development branch

### PR DESCRIPTION
**Pull request by dlebauer**
See #3

### CODE REVIEW

File: R/io.R [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    outlist <- fromJSON(df)
+    outdf <- tmpdf <- list()
+    for(site in names(outlist)){
+      for(pft in seq(length(outlist[[site]]))){
+        tmplist <- outlist[[site]][[pft]]
+        tmplist <- lapply(tmplist, function(df) ifelse(is.list(df), NA, df))
+        tmplist <- lapply(tmplist, function(df) ifelse(df == 0, NA, df))
+        tmplist[tmplist == "NaN"] <- NA
+        Location <- as.numeric(gsub("_data", "", gsub("site_", "", site)))
+        ## Temporary hack needs to be fixed
```
Comment (dlebauer) [inline R/io.R]: This will need to be fixed ...

File: src/ghgvc_script.R [hunk 2]
```diff
@@ -2,185 +2,94 @@
...
+##          config.list[[site]][[ecosystem.idx]]$latent %in% c(NA, 0)){
+#        config.list[[site]][[ecosystem.idx]]$latent <- biophys.defaults$latent[eidx]
+##     }
+#      
+#    }
+#  }
+#} 
+
+### Run the GHG calculator V2
 x <- ghgvc2(config.list)
```
Comment (dlebauer) [inline src/ghgvc_script.R]: note that `ghgvc2` is a hack function to clean up the output of `ghgvc`, but these should be merged as part of #3.

File: src/ghgvc_script.R [hunk 3]
```diff
@@ -2,185 +2,94 @@
...
-
-svgfixcmd <- paste("sed -i 's/symbol id/symbol overflow=\"visible\" id/g'", file.path(outdir, "output.svg"))
-system(svgfixcmd)
-#sub = annotate("text", x = 2, y = 0.3, parse = T, label = xlabels)))
+### Write the results to a file
+ghgcv_write(x, outdir, format = c("json", "csv"))
+
+
+### Create a plot of the results
+p = ghgcv_plot(x, outdir, config.list$options$T_E)
```
Comment (dlebauer) [inline src/ghgvc_script.R]: replace `=` with `<-` and, if you haven't done so yet, please review Hadley's style guide.

Comment (dlebauer) [inline src/ghgvc_script.R]: [Hadley's style guide](http://adv-r.had.co.nz/Style.html)

**Comment by potterzot** [comment]
Switch will be made. My preference for '=' as opposed to '<-' really betrays my python-first programming experience...

**Comment by dlebauer** [comment]
@potterzot can this be merged?

**Pull request merged**